src/async_fuse/memfs/kv_engine/mod.rs

```diff
@@ -225,8 +230,20 @@ pub trait KVEngine: Send + Sync + Debug + Sized {
     /// Lease grant
     async fn lease_grant(&self, ttl: i64) -> DatenLordResult<i64>;
 
+    /// Campaign leader
+    async fn campaign(&self, key: &KeyType, val: &ValueType, lease_id: i64) -> bool;
```

Comment:
Why not implement it using MetaTxn?

Reply:
I missed this part of the implementation, I'll fix it!